hack/test

```diff
@@ -38,22 +38,24 @@ function fmt_pass {
 		fi
 	done
 
+	checkPkgs=$(go list ./client/... ./cmd/... ./pkg/... ./test/e2e/...)
+	
 	go get honnef.co/go/tools/cmd/gosimple || true
 	if which gosimple >/dev/null; then
 		echo "Checking gosimple..."
-		checkRes=$(gosimple $(go list ./client/... ./cmd/... ./pkg/... ./test/e2e/...)) || true
-			if [ -n "${checkRes}" ]; then
-				echo -e "gosimple checking failed:\n${checkRes}"
-				exit 255
-			fi
+		checkRes=$(gosimple $checkPkgs)
```

Comment:
Yes I just double checked that removing the `|| true` will just cause the script to exit without printing if gosimple detects some improvements.

With `|| true`:
```
Haseebs-MacBook-Pro:etcd-operator haseeb$ PASSES="fmt" hack/test
Checking gofmt...
Checking govet...
Checking gosimple...
gosimple checking failed:
test/e2e/framework/framework.go:138:2: 'if err != nil { return err }; return nil' can be simplified to 'return err' (S1013)
```
Without `|| true`:
```
Haseebs-MacBook-Pro:etcd-operator haseeb$ PASSES="fmt" hack/test
Checking gofmt...
Checking govet...
Checking gosimple...
// gosimple output not printed
```